datadog_checks_base/tests/test_agent_check.py

```diff
@@ -3,6 +3,8 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+import sys
```

Comment:
is this import needed? (don't see it used anywhere)

Surprised a linter didn't catch this 🤔

Reply:
It was originally there when I was doing more mocking than necessary, will remove thanks for catching!